src/Traits/HasRelationships.php

```diff
@@ -8,13 +8,7 @@
 
 trait HasRelationships
 {
-    /**
-     * Retrieve the actual class name for a given morph class.
-     *
-     * @param  string  $class
-     * @return string
-     */
-    public static function getActualClassNameForMorph($class)
+    public static function getActualClassNameForMorph($class): string
```

Comment:
What is `$class`  a string? Could we get a type hint?

Reply:
I can't on this one because it extends a parent function :/